clearwater-socket-factory/clearwater_socket_factory.cpp

```diff
@@ -18,24 +18,24 @@
 #include <boost/algorithm/string/classification.hpp>
 #include <boost/algorithm/string/split.hpp>
 
-const char* MGMT_SOCKET_PATH = "/tmp/clearwater_management_namespace_socket";
-const char* SIGNALING_SOCKET_PATH = "/tmp/clearwater_signaling_namespace_socket";
 const int MAX_PENDING = 5;
 
-const char* DEFAULT_NS_FD_PATH = "/proc/1/ns/net";
-const std::string SIGNALING_NS_FD_DIR = "/var/run/netns/";
-
 const char* LOG_FILENAME = "/var/log/clearwater-socket-factory.log";
```

Comment:
I wasn't sure whether both instances should log to the same log file or not. On the one hand, interleaving logs from two different processes could make them harder to follow, and there is a small chance of a torn log (where one log line is written in the middle of another log). On the other hand, the socket factory isn't used very much (so the chances of interleaving logs is low), one log file is probably easier to manage, and you can separate the logs out after the fact using grep. Basically I don't care either way, but I wanted to check the two options have been considered :-).